l10n_ch_bank_statement_import_postfinance/static/src/js/account_statement_reconciliation.js

```diff
@@ -0,0 +1,18 @@
+/* This is Javascript extension of module account
```

Comment:
Add license in this header

https://github.com/OCA/maintainer-tools/blob/master/template/module/static/src/js/web_module_name.js#L1-L2